### DISCUSSION TITLE
[feature request] KukavarProxy

### DISCUSSION THREAD

**Discussion by ghost** [visose/Robots#100] [Q&A] [answered]
Hi Vicente,

Just wondering if you'd be willing to add support for KukavarProxy to robots? This will allow for "real-time" control and reading positions.

KukavarProxy:
https://github.com/ImtsSrl/KUKAVARPROXY

**Comment by visose** [comment, marked as answer]
Hi, I don't think I will take time to do this, but if you want to give it a go feel free to make a pull request.
